REST/PowerShell/Lifecycles/FetchAllLifecycleId.ps1

```diff
@@ -0,0 +1,7 @@
+$server = "https://<Your_Url>" #your octopus server
```

Comment:
This script lacks some of the common parts to our script examples, such as specifying a space name, retrieving the space id etc.

The variable `$command` name doesn't reflect what the string value is, its a combined uri